docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -1,48 +1,52 @@
+ As a general rule, add-ins should be treated as equivalent only when the following conditions are met, but there may be exceptional scenarios where add-ins are that don't meet these conditions can be treated as equivalent.
 
 - They're both created by the same developer, typically the entity specified in the [`"developer"`](/microsoft-365/extensibility/schema/root-developer) property of the unified manifest or the [ProviderName](/javascript/api/manifest/providername) element in the add-in only manifest.
 - They both are designed to be installed on the same Office application or applications, and they address the same workload with mainly overlapping functionality.
 - They have identical, or highly similar, public names, and user interfaces including control names and icons.
 
-If the Windows-only add-in is ever uninstalled, Office will automatically activate the Office Add-in the next time the host Office application is started.
 
 [!INCLUDE [Support note for equivalent add-ins feature](../includes/equivalent-add-in-support-note.md)]
 
-> [!TIP]
-> Currently, the Windows-only add-in is always given preference to the Office Add-in. But there's an enhancement to this feature in preview that provides you with a way to reverse this preference or to give the user a choice between the Windows-only add-in and Office Add-in. For more information, see [Option to disable the Windows-only add-in instead (preview)](#option-to-disable-the-windows-only-add-in-instead-preview).
+[!INCLUDE [new-outlook-vsto-com-support](../includes/new-outlook-vsto-com-support.md)]
+
+Because the Windows-only and the Office Add-in may not have exactly the same features, there are three scenarios you need to consider.
 
-## Specify an equivalent Windows-only add-in
+- Your Office Add-in does not provide all of the functionality that's available in the corresponding Windows-only add-in. In this scenario, you should [configure the Office add-in to be hidden](#hide-the-office-add-in) on any Windows computer on which the Windows-only add-in is installed. 
+- Your Office Add-in provides more and better functionality than the corresponding Windows-only add-in. In this scenario, you should [configure the Office add-in so that it detects the presence of the Windows-only add-in and hides it](#hide-the-windows-only-add-in).
```

Comment:
```suggestion
- Your Office Add-in provides more and better functionality than the corresponding Windows-only add-in. In this scenario, you should [configure the Office Add-in so that it detects the presence of the Windows-only add-in and hides it](#hide-the-windows-only-add-in).
```